File: reverse string/reverseString.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        # we can simply use s.reverse()
        j = len(s)-1
        for i in range(ceil(len(s)/2)):
            
            s[i],s[j] = s[j],s[i]
            j-=1

File: reverse string/reverseString.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        // we can simply use std::reverse function
        int i = 0;
        int j = s.size()-1;
        while (i < j)
        {
            char c = s[i];
            s[i]   = s[j];
            s[j]   =    c;
            i++;
            j--;
        }
    }
};
